util/test/chpl_launchcmd.py

```diff
@@ -619,6 +620,16 @@ def _cli_walltime(cls, walltime_str):
 
         raise ValueError('Did not recognize walltime: {0}'.format(walltime_str))
 
+    @classmethod
+    def _validate_args(cls, args, unparsed_args):
+        for arg in unparsed_args:
+            if re.search(r'-nl', arg):
```

Comment:
Think this is too broad and would catch things like `--my-arg-nl`. Probably watch `match()` to start at the beginning of the string (or just use `startswith`)

Reply:
Ugh, rushed the PR a bit too much. Updated.